src/main/java/org/openstreetmap/atlas/checks/atlas/predicates/TagPredicates.java

```diff
@@ -106,4 +107,17 @@
      */
     Predicate<AtlasObject> IS_CROSSING_HIGHWAY = object -> Validators.isOfType(object,
             HighwayTag.class, HighwayTag.CROSSING);
+
+    /**
+     * Tests if the {@link AtlasObject} has car navigable highway type and access tag not in Private
+     * enum set
+     */
+    Predicate<AtlasObject> IS_CAR_NAVIGABLE_NON_PRIVATE_HIGHWAY = object -> HighwayTag
+            .isCarNavigableHighway(object) && !AccessTag.isPrivate(object);
```

Comment:
I think if we're going to add something to a more general location like this, we should do the research to verify that it's the correct OSM definition.